Makefile

```diff
@@ -0,0 +1,29 @@
+# This Makefile is needed to compile the Rust glue layer between Devzat and the
+# Rust library Rustrict. To compile Devzat, simply run `make`. To install it as
+# a Go binary, run `make install`.
+
+all: devzat
+
+GO_SRC := colors.go commands.go config.go games.go main.go rpc.go slack.go twitter.go util.go
```

Comment:
This would include `censor.go` too, right? Is there a better way to do this: does `*.go` work?

Reply:
Well spotted. I will fix it.